runtime/arc.js

```diff
@@ -140,18 +138,18 @@ class Arc {
       if (!handleSet.has(handle.id))
         continue;
       let type = handle.type;
-      if (type.isSetView)
+      if (type.isCollection)
         type = type.primitiveType();
       if (type.isInterface) {
         interfaces += type.interfaceShape.toString() + '\n';
       }
       let key = this._storageProviderFactory.parseStringAsKey(handle.storageKey);
       switch (key.protocol) {
         case 'firebase':
-          handles += `view View${id++} of ${handle.type.toString()} '${handle.id}' @${handle._version} at '${handle.storageKey}'\n`;
+          handles += `store Store${id++} of ${handle.type.toString()} '${handle.id}' @${handle._version} at '${handle.storageKey}'\n`;
```

Comment:
I think you don't need the `.toString()` on `handle.type`, it's implicit.

Reply:
I think I'd rather keep it though, for clarity about what's happening.